### DISCUSSION TITLE
Documentation: Add instructions on updating Directus

### DISCUSSION THREAD

**Discussion by silasabbott** [directus/directus#6096] [Feature Requests]
I find little to no documentation on how to update Directus when self-hosted (with CLI or with One-Click deployment). It would be really helpful to add this to the documentation.

**Comment by rijkvanzanten** [comment]
https://docs.directus.io/guides/projects/#upgrading-a-project 🙂

**Reply by silasabbott** [reply]
Thanks! Would it be possible to link to this from the [Installation](https://docs.directus.io/getting-started/installation/) page? It's unintuitive to go to **Concepts** → **Projects** when looking to update Directus.

**Reply by rijkvanzanten** [reply]
Sure! Wanna open a PR for that?

**Reply by benhaynes** [reply]
I also think we should include the word "update" and "upgrade" in that headline so that it gets picked up by the search library.

**Reply by benhaynes** [reply]
https://github.com/directus/directus/pull/6110

**Reply by cstork** [reply]
I'm a npm newbie and just fell prey to this issue and spent 20min looking around. Sadly, #6110 did not solve the problem. See below why.

The first place I, and probably most people, will look for update instructions is at the end of the installation instructions. Please insert just a sentence linking to the maintenance/update instructions.

The second place is under guides. I'm sorry but it didn't occur to me to click on "Projects" to get to the updating/upgrading info.  There's also no guide listed for Installation (which I would have noticed) in the left-hand menu even though the URL https://docs.directus.io/guides/installation/cli/ indicates otherwise.

Besides this I spent this much time because unfortunately you'll have to search the documentation for exactly the term "updating". However, searching for "update" yields irrelevant results. So maybe add update to the search terms (or improve the search algo config). 

Send pull request #8023.

**Reply by benhaynes** [reply]
CC @alesvaupotic 

I agree we should add a notice referencing this at the end of installation, and should look into ways to make our headings more verbose, so they return better search results.

**Reply by cstork** [reply]
My PR uses the heading `## Configure / Update / Upgrade your Project`. So searching for "update" should then point you in the right direction.

**Comment by geantas** [comment]
New link to updating/upgrading documentation:
[https://docs.directus.io/configuration/upgrades-migrations/](https://docs.directus.io/configuration/upgrades-migrations/ )
